gulpfile.babel.js

```diff
@@ -34,7 +36,14 @@ gulp.task('lint', () => {
     .pipe(plugins.eslint.format());
 });
 
-gulp.task('test', ['lint']);
+gulp.task('test:mocha', ['build'], () => {
+  const tf = yargs.option('tf', { alias: 'testFilename', default: 'test/test*.js', type: 'string' }).argv.tf;
```

Comment:
Missed it earlier: Could you move the `const argv = yargs.option(…` in the header of the file?